Tests/SwiftParserTest/IncrementalParsingTests.swift

```diff
@@ -16,31 +16,85 @@ import SwiftParser
 
 public class IncrementalParsingTests: XCTestCase {
 
+  public func testDiffOfTwoStringsSimple() {
+    let s1 = "struct A { func f() {"
+    let s2 = "struct AA { func f() {"
+
+    let diffs = getDiffOfTwoString(s1, s2)
+    XCTAssert(diffs.count == 1)
+
+    let firstDiff = diffs.first!
+    XCTAssert(firstDiff == SourceEdit(offset: 8, length: 0, replacementLength: 1))
+  }
+
+  public func testDiffOfTwoSameStrings() {
+    let s1 = "0123456"
+    let s2 = "0123456"
+
+    let diffs = getDiffOfTwoString(s1, s2)
+    XCTAssert(diffs.count == 0)
+  }
+
+  public func testDiffOfTwoStrings() {
+    let s1 = "0123456"
+    let s2 = "x12456yz"
+
+    // should return [(0, 1, 0), (0, 0, 1), (3, 1, 0), (6, 0, 1), (7, 0, 1)]
+    let diffs = getDiffOfTwoString(s1, s2)
+
+    let expectedDiffs: [SourceEdit] = [
+      SourceEdit(offset: 0, length: 1, replacementLength: 0),
+      SourceEdit(offset: 0, length: 0, replacementLength: 1),
+      SourceEdit(offset: 3, length: 1, replacementLength: 0),
+      SourceEdit(offset: 6, length: 0, replacementLength: 1),
+      SourceEdit(offset: 7, length: 0, replacementLength: 1),
+    ]
+
+    XCTAssert(diffs == expectedDiffs)
```

Comment:
I think `applyEdits` is written in `SwiftSyntaxTest` and we're in `SwiftParserTest` here so I could not directly call it. If that's necessary I could move it to elsewhere.

Reply:
Actually, now that I think about it, it would make sense to move `getConcurrentEdits` to `_SwiftSyntaxTestSupport` and move all of your new tests in `Tests/SwiftSyntaxTestSupportTests` because they are really testing that this helper function actually works. You can then also move `applyEdits` to `_SwiftSyntaxTestSupport` and use it from your new tests.